code/__HELPERS/priority_announce.dm

```diff
@@ -49,13 +49,16 @@
 			C.messagetext.Add(text)
 			P.update_icon()
 
-/proc/minor_announce(message, title = "Attention:", alert)
+/proc/minor_announce(message, title = "Attention:", alert, mob/living/from)
```

Comment:
Make this one just a `/mob/`; admin ghosts also have the ability to use those, theoretically.